slack/main.go

```diff
@@ -108,9 +108,12 @@ func (s *slackNotifier) writeMessage() (*slack.WebhookMessage, error) {
 	var clr string
 	switch build.Status {
 	case cbpb.Build_SUCCESS:
-		clr = "good"
-	case cbpb.Build_FAILURE, cbpb.Build_INTERNAL_ERROR, cbpb.Build_TIMEOUT:
-		clr = "danger"
+		// https://stackoverflow.com/q/70039333
+		clr = "#2EB67D"  // green
```

Comment:
I don't know why passing names of color (e.g. "good") doesn't work well.
It seems to accept only hex values.

And I defined thes colors via https://usbrandcolors.com/slack-colors/.